cluster/cluster.go

```diff
@@ -482,7 +451,7 @@ func DeleteTenantFolderInDisk(tenant *Tenant, sg *StorageGroup, sgNode *StorageG
 
 			newFolderForDeletion := fmt.Sprintf("%s/%s-to-delete-%s", vol.MountPath, tenant.ShortName, randSringForDeletion)
 			// move current tenant path for one to be deleted and delete if afterwards
-			commands = append(commands, fmt.Sprintf(`mv -v %s/%s %s && rm -rv %s`, vol.MountPath, tenant.ShortName, newFolderForDeletion, newFolderForDeletion))
+			commands = append(commands, fmt.Sprintf(`mv -v %s/%s %s && rm -rv %s && mkdir -p %s/%s`, vol.MountPath, tenant.ShortName, newFolderForDeletion, newFolderForDeletion, vol.MountPath, tenant.ShortName))
```

Comment:
Even if this values were auto generated (tenant shortname, etc) we need to validate all the parameters that goes into exec to minimize the possibility of command injection vulnerabilities via a previous sql injection, we can use regex for that https://github.com/minio/m3/commit/43b37eaba1784a8d1514dfc1959f9b53ea6bf704#diff-535fa68c8799a7d7e71f34e59b8c34faR59, please validate that provided vol.MountPath, tenant.ShortName, newFolderForDeletion, newFolderForDeletion, vol.MountPath, tenant.ShortName consist only of `[a-z]` characters

Reply:
Or avoid all of this altogether and use helpers like renameAll and removeAll written in cmd/os-reliable.go in `minio/minio` project @cesnietor

Reply:
since this is just a job that is running a simple cli command on an Ubuntu Image. I think we can keep it like that, else it would need to have a whole go project just removing a folder, I added a validation of the tenant shortname to prevent it from being more than a valid name.  let me know your thoughts.